ItsME/Presentation/Scenes/Home/HomeViewController.swift

```diff
@@ -130,30 +130,34 @@ private extension HomeViewController {
     
     var userInfoBinding: Binder<UserInfo> {
         return .init(self) { viewController, userInfo in
-            userInfo.defaultItems.forEach { item in
-                let profileInfo: ProfileInfoComponent = .init(userInfoItem: item)
-                self.vStackLayout.addArrangedSubview(profileInfo)
-            }
-            self.vStackLayout.snp.makeConstraints { make in
-                make.height.equalTo((self.vStackLayout.arrangedSubviews.count * 40))
-                make.width.equalTo(300)
-                make.centerX.equalTo(self.view.safeAreaLayoutGuide)
-                make.top.equalTo(self.profileImageView.snp.bottom).offset(20)
+            if self.vStackLayout.arrangedSubviews.isEmpty{
```

Comment:
```suggestion
            if self.vStackLayout.arrangedSubviews.isEmpty {
```
아래 if문 처럼 space를 줬습니다.